Controller.cs

```diff
@@ -69,111 +89,110 @@ public static class Controller {
 
         }
 
-        public static bool IsPwmFromString(string pin) {
-            pin = pin.ToLower();
-
+        public static bool IsPwmFromPin(double pin) {
             if (IsPulse) {
                 switch (pin) {
-                    case "p3":
-                    case "p0":
-                    case "p1":
-                    case "p8":
-                    case "p12":
-                    case "p2":
+                    case 3:
```

Comment:
It would be better to use the constants defined earlier

Reply:
Done.